h/static/scripts/directives/markdown.coffee

```diff
@@ -336,7 +336,7 @@ markdown = ['$filter', '$sanitize', '$sce', '$timeout', ($filter, $sanitize, $sc
     # React to the changes to the input
     inputEl.bind 'blur change keyup', ->
       ctrl.$setViewValue inputEl.val()
-      scope.$digest()
+#      scope.$digest()  # TODO: this shouldn't be a problem
```

Comment:
> I had to temporarly comment this out, because I was getting "already in $digest" errors. 

This is the only thing left to solve in this PR.
Anybody feels like debugging this?